app/templates/buckets/details.html

```diff
@@ -1,45 +1,49 @@
 {% extends "layouts/one-column.html" %}
 
-{% set page_title = "Bucket: " + bucket.name %}
+{% set page_title = "App data source: " + bucket.name %}
 
 {% block main_column %}
-<!-- TODO: This is only a placeholder template - Update me -->
 
 
-<!-- TODO: Placeholder list - double check wording and everything -->
-<h2 class="heading-medium">Users with access</h2>
-<p>{{ bucket.users3buckets.length }} user{%- if bucket.users3buckets.length != 1 -%}s have{% else %} has{% endif %} access to this data source</p>
+<h2 class="heading-medium">Users with admin access</h2>
+<p>{{ bucket.users3buckets.length }} user{%- if bucket.users3buckets.length != 1 -%}s are{% else %} is{% endif %} admin of this app data source</p>
 
 {% if bucket.users3buckets.length %}
   <table class="form-group">
     <thead>
       <tr>
         <th>User</th>
         <th>User has read/write access</th>
-        <th></th>
+        <th>
+          {% if current_user_is_bucket_admin or current_user.is_superuser %}
+            <span class="visuallyhidden">Actions</span>
+          {% endif %}
+        </th>
       </tr>
     </thead>
     <tbody>
       {% for users3bucket in bucket.users3buckets %}
         <tr>
-          <td><a href="{{ url_for('users.details', {id: users3bucket.user.auth0_id}) }}">{{ users3bucket.user.username }}</a></td>
+          <td><a class="{% if current_user_id == users3bucket.user.auth0_id %}highlight-current{% endif %}" href="{{ url_for('users.details', {id: users3bucket.user.auth0_id}) }}">{{ users3bucket.user.name }}</a></td>
```

Comment:
You might be able to change this to `if current_user == users3bucket.user`

Reply:
Is that better?

Reply:
Also, that doesn't work as `users3bucket.user` is a vastly reduced set of data.

Reply:
👍